File: POTD/potd(2-10-2023)/potd(2-10-2023).py
class Solution:
    def winnerOfGame(self, colors: str) -> bool:
        a,b = 0,0
        l = len(colors)
        for i in range(1, l-1):
            if(colors[i] == "A" and colors[i+1] == "A" and colors[i-1 ] == "A"):
                a += 1
            elif(colors[i] == "B" and colors[i+1] == "B" and colors[i-1 ] == "B"):
                b += 1
        return a>b

File: POTD/potd(2-10-2023)/potd(2-10-2023).cpp
class Solution
{
public:
    bool winnerOfGame(string colors)
    {
        int a = 0;
        int b = 0;
        for (int i = 1; i < colors.length() - 1; i++)
        {
            char currentChar = colors[i];
            if (currentChar == 'A' && colors[i - 1] == 'A' && colors[i + 1] == 'A')
            {
                a++;
            }
            else if (currentChar == 'B' && colors[i - 1] == 'B' && colors[i + 1] == 'B')
            {
                b++;
            }
        }

        return a > b;
    }
};